packages/retail-ui-extensions-react/src/extension-api/connectivity-api/connectivity-api.ts

```diff
@@ -0,0 +1,72 @@
+import type {StatefulRemoteSubscribable} from '@shopify/retail-ui-extensions';
+import {makeStatefulSubscribable} from '@shopify/retail-ui-extensions';
+import {useExtensionApi} from '../utils';
+import {useEffect, useRef, useState} from 'react';
+import {ConnectivityState} from '@shopify/retail-ui-extensions/src/extension-api/connectivity-api/connectivity-api';
```

Comment:
You should be able to import this from `@shopify/retail-ui-extensions`, so there's an export chain missing somewhere.

Reply:
Thanks!